client/src/assets/Keywords/index.js

```diff
@@ -0,0 +1,33 @@
+import family from './family.png';
+import solo from './solo.png';
+import wild from './wild.png';
+// import commitment from './commitment.png';
+// import interaction from './interaction.png';
+// import kid from './kid.png';
+// import immersion from './immersion.png';
+// import luxury from './luxury.png';
+// import nature from './nature.png';
+// import rural from './rural.png';
+// import social from './social.png';
+// import staff from './staff.png';
+// import urban from './urban.png';
+// import vegan from './vegan.png';
+// import vegetarian from './vegetarian.png';
+
+export default {
+  solo,
+  wild,
+  family,
+  //   commitment,
+  //   interaction,
+  //   kid,
+  //   immersion,
+  //   luxury,
+  //   nature,
+  //   rural,
+  //   social,
+  //   staff,
+  //   urban,
+  //   vegan,
+  //   vegetarian,
+};
```

Comment:
I would get rid of this file (at the very least, it shouldn't be in the assets folder), see my comment for the KeywordsData file

Reply:
We will decide on it again, later when we figure where are we gonna store the images, for now they are here, as a minimum of 3 pngs
about the structure i think its fine ?